.github/workflows/main-release.yaml

```diff
@@ -57,6 +57,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           COSIGN_PASSWORD: ${{ secrets.COSIGN_PASSWORD }}
           ACORN_DNS_ENDPOINT: https://staging-dns.acrn.io/v1
+          LE_ENDPOINT: https://acme-staging-v02.api.letsencrypt.org/directory
```

Comment:
this and the next two files should be unnecessary now, right?

Reply:
Correct, removed.